test/nri/plugin.go

```diff
@@ -617,6 +637,10 @@ func RunPodEvent(pod string) *event {
 	return PodEvent("RunPodSandbox", pod)
 }
 
+func UpdatePodEvent(pod string) *event {
+	return PodEvent("UpdatePodSandbox", pod)
+}
+
```

Comment:
Eventually we should update this to also record the updated overhead and resources fields, so we can have more meaningful tests.